### PR TITLE
[stoney] Make adding a node to a cluster give all roles from that cluster to node

### PR DESCRIPTION
**Pull request by vuntz**
This is https://github.com/crowbar/barclamp-pacemaker/pull/157 for stoney. It required a small change.

### CODE REVIEW

**Comment by jsuchome** [comment]
I prepared 2 nodes cluster with keystone and glance. Later I've added 3rd node, and pacemaker barclamp applied successfully. All needed primitives were created.

File: crowbar_framework/app/models/pacemaker_service.rb [hunk 1]
```diff
@@ -75,6 +75,150 @@ def create_proposal
...
+
+    # Find all barclamp roles where this cluster is used
+    cluster_roles = RoleObject.all.select do |role_object|
+      role_object.name.to_s =~ /.*\-config\-.*/ && \
+      role_object.elements.values.flatten.compact.uniq.include?(cluster_element)
+    end
+
+    # Inside each barclamp role, identify which role is required
+    for cluster_role in cluster_roles do
+      service = ServiceObject.get_service(cluster_role.barclamp).new(Rails.logger)
```
Comment (jsuchome) [inline crowbar_framework/app/models/pacemaker_service.rb]: Why is Rails.logger argument here?

**Comment by jsuchome** [comment]
The code looks good, +1 from me

File: crowbar_framework/app/models/pacemaker_service.rb (hunk 1 above)
Comment (vuntz) [inline crowbar_framework/app/models/pacemaker_service.rb]: You need to give a logger when creating a service instance.

**Comment by dirkmueller** [comment]
+1

**Pull request merged**